File: src/main/java/conditionalStatements/ThreeDices.java
package conditionalStatements;

import java.util.Scanner;

public class ThreeDices {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int first = scanner.nextInt();
        int second = scanner.nextInt();
        int third = scanner.nextInt();
        scanner.close();

        if (first == second && first == third) {
            System.out.println(10_000 + first * 1_000);
        } else if (first == second) {
            System.out.println(1_000 + first * 100);
        } else if (first == third) {
            System.out.println(1_000 + first * 100);
        } else if (second == third) {
            System.out.println(1_000 + second * 100);
        } else {
            System.out.println(Math.max(first, Math.max(second, third)) * 100);
        }
    }

}

File: src/main/kotlin/conditionalStatements/ThreeDices.kt
package conditionalStatements

import java.util.Scanner
import kotlin.math.max

fun main() {
    val scanner = Scanner(System.`in`)
    val first: Int = scanner.nextInt()
    val second: Int = scanner.nextInt()
    val third: Int = scanner.nextInt()
    scanner.close()

    when {
        first == second && second == third -> println(10_000 + first * 1_000)
        first == second -> println(1_000 + first * 100)
        first == third -> println(1_000 + first * 100)
        second == third -> println(1_000 + second * 100)
        else -> println(max(first, max(second, third)) * 100)
    }
}